utils/frame/remote-externalities/src/lib.rs

```diff
@@ -161,7 +175,13 @@ pub struct OnlineConfig {
 impl Default for OnlineConfig {
 	fn default() -> Self {
 		Self {
-			uri: "http://localhost:9933".into(),
+			uri: TARGET.to_owned(),
+			rpc: Arc::new(
+				HttpClient::new(
```

Comment:
This is wrong -- We can't initialize the client here. Then, if we override the `at` later, if will be a noop. Will push fixes.

Reply:
(or maybe it is correct -- have to check)

Reply:
ok, I didn't understand that the URL/URI could be replaced thanks for the fix.